src/BuildScriptGeneratorCli/BuildScriptGeneratorCli.csproj

```diff
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
-    <RuntimeFrameworkVersion>2.1</RuntimeFrameworkVersion>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Can you create new projects in 3.1 and see what new xml elements they generate and have them similarly here? Idea is to avoid having elements which might not make sense any longer.

Reply:
Also we need to place a request with the DDIT team to upgrade the .NET Core SDK to 3.1 version first otherwise your PR build will not succeed.